app/src/main/java/bomb/AtomicBomb.java

```diff
@@ -0,0 +1,44 @@
+package bomb;
+import android.graphics.Paint;
+
+import java.util.List;
+import hacking.to.the.gate.CircleCollider;
+import hacking.to.the.gate.Hittable;
+import hacking.to.the.gate.Position;
+
+/**
+ * Created by Ruiqian on 9/12/2015.
+ */
+public class AtomicBomb extends Bomb{
+
+    private BombLifeCycle bombLifeCycle;
```

Comment:
well, because only atomic bomb triggers vibration currently, but yes, it's more reasonable to put it in Bomb class